### ISSUE TITLE
Calculate eigenvectors numerically if it is impossible to calculate them symbolically

### ISSUE THREAD

**Issue by goodok** [sympy/sympy#6288]
```
Try to calculate eigenvectors numericly, if it impossible calculate symbolical.

As there are cases when http://code.google.com/p/sympy/issues/detail?id=2193
```

Original issue for #6288: http://code.google.com/p/sympy/issues/detail?id=3189
Original author: https://code.google.com/u/109448925098397033296/

**Comment by smichr** [comment]
[Matrices containing imaginary numbers](https://github.com/sympy/sympy/issues/6288#issuecomment-37005379) appear to work now:

```
>>> Matrix(2, 2, [1, 0, 0, I]).eigenvects()
[(1, 1, [Matrix([
[1],
[0]])]), (I, 1, [Matrix([
[0],
[1]])])]
```

**Comment by sylee957** [comment]
This was resolved in #18911
